wemake_python_styleguide/visitors/ast/naming.py

```diff
@@ -126,6 +128,38 @@ def visit_Name(self, node: ast.Name) -> None:
 
         self.generic_visit(node)
 
+    # FIXME: Remove "noqa" after fixing problem with comprehension
+    # variables being counted as locals.
+    def visit_ClassDef(self, node: ast.ClassDef) -> None:  # noqa: Z210
+        """
+        Used to find upper attribute declarations.
+
+        Raises:
+            WrongVariableNameViolation
+            TooShortVariableNameViolation
+            PrivateNameViolation
+
+        """
+        top_level_assigns = [
```

Comment:
Can you please move all the business logic into a separate method?

Reply:
I have considered your suggestion, but I prefer to be consistent.

Reply:
So I'll just leave `visit_ClassDef` around call to that separate method?